app/src/main/java/ru/thstdio/aa2020/ui/detail/FragmentMoviesDetails.kt

```diff
@@ -88,9 +77,13 @@ class FragmentMoviesDetails : FragmentNavigation(R.layout.fragment_movies_detail
         }
     }
 
-    override fun onDestroyView() {
-        scope.cancel()
-        super.onDestroyView()
-
+    private fun showActorList(actors: List<Actor>) {
+        if (actors.isNotEmpty()) {
+            val adapter = binding.recyclerView.adapter as ActorAdapter?
```

Comment:
use not
as ActorAdapter?
but
as? ActorAdapter

Reply:
fix